packages/design-system/src/components/TextField/TextField.test.tsx

```diff
@@ -1,27 +1,28 @@
-import { TextField, unmaskValue } from './TextField';
-import { mount, shallow } from 'enzyme';
 import React from 'react';
+import { TextField, unmaskValue } from './TextField';
+import { render, screen } from '@testing-library/react';
+import { DATE_MASK } from './useLabelMask';
 
-function render(customProps = {}, deep = false) {
-  const props = Object.assign(
-    {
-      label: 'Foo',
-      name: 'spec-field',
-    },
-    customProps
-  );
-  const component = <TextField {...props} />;
+const defaultProps = {
+  label: 'Foo',
+  name: 'spec-field',
+};
 
-  return {
-    props: props,
-    wrapper: deep ? mount(component) : shallow(component),
-  };
+function renderTextField(customProps = {}) {
+  return render(<TextField {...defaultProps} {...customProps} />);
 }
 
 describe('TextField', function () {
   it('renders', () => {
-    const data = render();
-    expect(data.wrapper).toMatchSnapshot();
+    expect(renderTextField().asFragment()).toMatchSnapshot();
+  });
+
+  it('renders with a label mask', () => {
+    expect(renderTextField({ labelMask: DATE_MASK }).asFragment()).toMatchSnapshot();
```

Comment:
I considered not using snapshots, but any meaningful queries I could make against these two other cases would expose implementation details and be more work to maintain. I figured since the snapshots are fairly small, they're more maintainable than the alternative.

Reply:
Seems like a good use-case for snapshots